### PR TITLE
Revamp of the wings stuff.

### PR DESCRIPTION
**Pull request by KaliLusternia**
It now runs on the mmp link externals and mmp clear externals events, instead of being copy-pasted in three or four different spots
Slimmed down considerably, and got rid of that pile of local variables, where there was one for every exit created. 
Changed the way the gone-outside script works. Now, instead of looking for one of a set of commands that it associates with a wings exit, it'll check if the path it created to check for a shorter path would take it in a different direction than the previously calculated path. 
Added in three more transport devices for lusternia, and set up their configs and things.
Also sorted the private settings table into groups because looking through it to check for particular things was annoying me to no end.
I've not been able to test the achaea things thoroughly, but I don't think I broke anything there. - I was able to test with eagle wings, but none of the other wing types or the harness.

### CODE REVIEW

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -4895,6 +4918,8 @@ toycurio={lusternia=true},
 utensilcurio={lusternia=true},
 vernalcurio={lusternia=true},
 wheel={lusternia=true},
+mantle={lusternia=true},
+key={lusternia=true},
```
Comment (keneanung) [inline mudlet-mapper.xml]: I believe the soulless curio is missing in this filter list.

File: mudlet-mapper.xml [hunk 2]
```diff
@@ -5815,6 +5515,229 @@ end</script>
...
+                <script>local exitsCreated = exitsCreated or {}
+
+local function tempSpecialExit(destination, command)
+  table.insert(exitsCreated, destination)
+  addSpecialExit(mmp.currentroom, destination, command)
+end
+
+function mmp.addWings()
+--Trimmed down version of the original function, meant to only be used for wings. (Where the command varies based on several MCONFIGS)
+  local function getcmd(word)
```
Comment (keneanung) [inline mudlet-mapper.xml]: The formatting below looks funny, at least in this diff... Maybe consider using `string.format`?

File: mudlet-mapper.xml [hunk 3]
```diff
@@ -5815,6 +5515,229 @@ end</script>
...
+      word ..
+      [[&quot;, &quot;remove wings&quot;, false)]] or
+      [[script:send(&quot;say *]] ..
+      (mmp.settings.winglanguage and mmp.settings.winglanguage or &quot;&quot;) ..
+      [[ ]] ..
+      word ..
+      [[&quot;, false)]]
+  end
+
+  --creates a special exit from your current room to a destination, and adds it to a table so it can be easily cleared later.
```
Comment (keneanung) [inline mudlet-mapper.xml]: I wonder if it's not better to use one event handler per game instead of having one big for each.

**Comment by KaliLusternia** [comment]
Hopefully that commit should take care of all those issues!

**Comment by vadi2** [comment]
Nice, thanks for revamping this! Things got a bit out of hand with people copy/pasting the initial solution for everything else, so this needed to happen.

**Pull request merged**